dashbuilder/dashbuilder-client/dashbuilder-cms-client/src/main/java/org/dashbuilder/client/cms/resources/i18n/ContentManagerConstants.java

```diff
@@ -20,7 +20,7 @@
 
 public interface ContentManagerConstants extends Messages {
 
-    ContentManagerConstants INSTANCE = GWT.create(ContentManagerConstants.class);
+    public static final ContentManagerConstants INSTANCE = GWT.create(ContentManagerConstants.class);
```

Comment:
Once again, no need for modifiers

Reply:
I agree.
Several interfaces use this model though (see for example UberfireConstants.java)
Do you want me to change like in AppResource.java ?

Reply:
Yes, I remember seeing it in some classes. I don't know what's the reason behind using this like that and if we need to follow that convention. We can keep it as is but this might be a good candidate for a further discussion with others (for future PRs).